### PR TITLE
Implement CRUD

### PR DESCRIPTION
**Pull request by swengineermwi**
- Add delete
- Add remove
- Update item index to match array length

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    setTimeout(() => {
+      btn.removeEventListener('click', () => { })
+    }, 2000)
+  }
+}
+
+export function clear() {
+  let tasks = JSON.parse(localStorage.getItem('things'))
+  if (tasks !== null && tasks.length > 0) {
+    tasks = tasks.filter(check)    // Returns [32, 33, 40]
```
Comment (bellom) [inline src/crud.js]: Kindly remove commented code. `// Returns [32, 33, 40]`
It’s better to delete the commented code. If you need to review a previous version of your code, use git history.

File: src/index.js [hunk 2]
```diff
@@ -6,94 +6,101 @@ import { dragover } from './draggability';
...
 
-    g('todo_list').appendChild(tasktemplate);
+    g('clear').addEventListener('click', clear)
   }
 }
 
-onload = loadTodo();
+onload = updateIndices(() => { loadTodo() });
 /* eslint-disable no-plusplus */
 /* eslint-disable no-restricted-globals */
```
Comment (bellom) [inline src/index.js]: With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @softwareengineermwi  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and the necessary methods  🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

1. _It is always professional to commit often, commit every new feature or method added._

2. _Kindly check the comments under the review._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (swengineermwi) [reply src/index.js]: Hi @bellom, I have made the required changes. Please check.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @softwareengineermwi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations once again! 🎉 



### Keep Rocking!👏👏👏

**Pull request merged**